### ISSUE TITLE
Create right html structure for NcBreadcrumbs.vue and NcBreadcrumb.vue

### ISSUE THREAD

**Issue by JuliaKirschenheuter** [nextcloud/nextcloud-vue#3951]
Related ticket: https://github.com/nextcloud/server/issues/36918

The breadcrumb navigation represents a list and should best be implemented as a list.

To be fixed in:

- [ ] https://github.com/nextcloud/nextcloud-vue/blob/master/src/components/NcBreadcrumbs/NcBreadcrumbs.vue
- [ ] https://github.com/nextcloud/nextcloud-vue/blob/master/src/components/NcBreadcrumb/NcBreadcrumb.vue

TODO:

- [ ] styles for narrow screens: when the screen is too narrow, the menu appear and breadcrumbs items should be hidden.
- [ ] html structure: list which is inside of the `nav` navigation.
- [ ] screen reader visibility for narrow screens: when the screen is too narrow, the breadcrumbs items and the actions menu should be hidden (`aria-hidden="true"`).

See the right way of implementation in: https://github.com/nextcloud/server/issues/36918

**Comment by JuliaKirschenheuter** [comment]
@skjnldsv Am i right with ticket description?

**Comment by skjnldsv** [comment]
> @skjnldsv Am i right with ticket description?

I'm confused by the TODO section phrazing

TODO:
- [ ] Make sure screen readers do not read the actions menu content
- [ ] Make sure all breadcrumbs are still rendered on narrow screen, but not visible so screen readers still understand the NAV structure
- [ ] Make sure hidden breadcrumbs are not interractable (is that even a word?) so that nav semantic is respected, but the keyboard navigation is still correct

**Comment by raimund-schluessler** [comment]
PR is in #3990.
